pom.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
+    <artifactId>selenium-ta-project</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+        <org.apache.maven.plugins>2.22.2</org.apache.maven.plugins>
+        <defaultSuite>debug</defaultSuite>
+        <suite>${defaultSuite}</suite>
+        <defaultEnv>dev</defaultEnv>
+        <env>${defaultEnv}</env>
+
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <java.version>17</java.version>
+<!--        <selenium.version>3.141.59</selenium.version>-->
+        <webdrivermanager.version>5.1.1</webdrivermanager.version>
+        <testng.version>7.5</testng.version>
+        <assertj-core.version>3.22.0</assertj-core.version>
+        <log4j-core.version>2.17.2</log4j-core.version>
+        <log4j-api.version>2.17.2</log4j-api.version>
+        <slf4j-simple.version>1.7.7</slf4j-simple.version>
+        <allure-testng.version>2.17.3</allure-testng.version>
+        <allure-maven.version>2.11.2</allure-maven.version>
+        <aspectj.version>1.9.8</aspectj.version>
+        <suiteXmlFile>src/test/java/tests/suites/${defaultSuite}.xml</suiteXmlFile>
+        <webdrivermanager.version>5.1.1</webdrivermanager.version>
+        <extentreports.version>4.0.0</extentreports.version>
+        <maven-compiler-plugin.version>3.2</maven-compiler-plugin.version>
+        <allure.version>2.17.1</allure.version>
+        <allure.results.directory>${basedir}/target/allure-results</allure.results.directory>
+    </properties>
+    <dependencies>
+        <!--<dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-java</artifactId>
+            <version>${selenium.version}</version>
+        </dependency>-->
+        <dependency>
```

Comment:
please remove selenide contains this transitive dependency

Reply:
removed selenium